src/lib.rs

```diff
@@ -221,6 +221,7 @@ pub async fn start_lemmy_server(args: CmdArgs) -> Result<(), LemmyError> {
       federation_config.clone(),
     )
   });
+  panic!("eeeeeeeeeee");
```

Comment:
I think this panic prevents api tests from starting so it keeps running until the ci timeout. You need to put the panic inside some http handler.